include/vast/Dialect/HighLevel/HighLevelCF.td

```diff
@@ -79,7 +79,7 @@ def HighLevel_TypeYieldOp : HighLevel_Op< "type.yield", [
   let assemblyFormat = [{ attr-dict $result `:` type($result) }];
 }
 
-def HighLevel_IfOp : ControlFlowOp< "if" >
+def HighLevel_IfOp : ControlFlowOp< "if", [DeclareOpInterfaceMethods<RegionKindInterface>] >
```

Comment:
Cn not `[DeclareOpInterfaceMethods<RegionKindInterface>]` be part of `ControlFlowOp`?